keyboards/keebio/iris/readme.md

```diff
@@ -3,18 +3,36 @@ Iris
 
 A split ergo 4x6 keyboard with 3 or 4 thumb keys made and sold by Keebio. [More info at Keebio](https://keeb.io).
 
-Keyboard Maintainer: [Bakingpy/nooges](https://github.com/nooges)  
-Hardware Supported: Pro Micro  
-Hardware Availability: [Keebio](https://keeb.io)  
+Keyboard Maintainer: [Bakingpy/nooges](https://github.com/nooges)
+Hardware Supported: Iris 3 PCB, Pro Micro, and Elite-C
+Hardware Availability: [Keebio](https://keeb.io)
+
+### Revision 3
 
 Make example for this keyboard (after setting up your build environment):
 
-    make keebio/iris/rev2:default
+    make keebio/iris/rev3:default
 
 Example of flashing this keyboard:
 
+    make keebio/iris/rev3:default:dfu
+
+### Revision 2
+
+Make example for this keyboard (after setting up your build environment):
+
+    make keebio/iris/rev2:default
+
+Example of flashing this keyboard (Pro Micro):
+
     make keebio/iris/rev2:default:avrdude
 
+Example of flashing this keyboard (Elite-C):
+
+    make keebio/iris/rev2:default:dfu
+
 See [build environment setup](https://docs.qmk.fm/#/newbs_getting_started) then the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information.
```

Comment:
Might as well also update this line to the wording currently used in the template.

```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```